logging/bin/deploy_fluentbit_open.sh

```diff
@@ -92,7 +91,7 @@ else
    # use copy in repo
    FB_CONFIGMAP="logging/fb/fluent-bit_config.configmap_open.yaml"
 fi
-log_info "Using FB ConfigMap:" $FB_CONFIGMAP
+# log_info "Using FB ConfigMap:" $FB_CONFIGMAP
 
```

Comment:
The message should be written out at some level...probably log_verbose or log_debug.

Reply:
I went ahead and updated this to DEBUG to align with the other messages that reference a specific file in the USER_DIR directory.